src/math/ColorManagement.js

```diff
@@ -76,13 +76,13 @@ function createColorManagement() {
 
 		},
 
-		fromWorkingColorSpace: function ( color, targetColorSpace ) {
+		workingToColorSpace: function ( color, targetColorSpace ) {
```

Comment:
Looking at addon related change, these methods might be potentially used in third party code.

Maybe it's more safe to keep both methods but add deprecation warnings so we can remove them in ten releases. Internally, they call the new methods.

Reply:
@Mugen87 Can you please advise as to the pattern you want to use for deprecation here?

Or merge this as-is, and add it in a separate PR.

Reply:
Maybe like so:
```js
fromWorkingColorSpace: function ( color, targetColorSpace ) {

	warnOnce( 'THREE.ColorManagement: .fromWorkingColorSpace() has been renamed to .workingToColorSpace().' ); // @deprecated, r177

	return ColorManagement.workingToColorSpace( color, targetColorSpace );

},

toWorkingColorSpace: function ( color, targetColorSpace ) {

	warnOnce( 'THREE.ColorManagement: .toWorkingColorSpace() has been renamed to .colorSpaceToWorking().' ); // @deprecated, r177

	return ColorManagement.colorSpaceToWorking( color, targetColorSpace );

},
```